Fails to compile

**Description of the bug**
```
$ make
-e \e[36;1m*** Building debug target...\e[0m
    Updating crates.io index
  Downloaded failure v0.1.6
  Downloaded subtle v2.2.1
  Downloaded failure_derive v0.1.6
  Downloaded synstructure v0.12.3
  Downloaded zeroize v0.6.0
  Downloaded zeroize_derive v0.1.0
  Downloaded syn v0.15.44
   Compiling proc-macro2 v1.0.6
   Compiling proc-macro2 v0.4.30
   Compiling unicode-xid v0.2.0
   Compiling unicode-xid v0.1.0
   Compiling syn v1.0.8
   Compiling syn v0.15.44
   Compiling failure_derive v0.1.6
   Compiling subtle v2.2.1
   Compiling quote v0.6.13
   Compiling quote v1.0.2
   Compiling zeroize_derive v0.1.0
   Compiling synstructure v0.12.3
   Compiling zeroize v0.6.0
   Compiling failure v0.1.6
   Compiling deoxysii v0.2.0 (/Users/ur20980/src/deoxysii-rust)
error: The following target_feature flags must be set: +aes,+ssse3.
  --> src/lib.rs:28:1
   |
28 | compile_error!("The following target_feature flags must be set: +aes,+ssse3.");
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

error: could not compile `deoxysii`.

To learn more, run the command again with --verbose.
make: *** [debug] Error 101
$ 
```

**Steps to reproduce**
1. `git clone https://github.com/oasislabs/deoxysii-rust.git`
2. `cd deoxysii-rust`
3. `make`

**Expected behavior**
At the very least, a successfully built crate.

**Environment**
 - OS: macOS Mojave 10.14.6.
 - `rustc 1.41.0-nightly (0f0c640e0 2019-11-17)` and `rustc 1.39.0 (4560ea788 2019-11-04)`
- CPU: 3 GHz Intel Xeon W (`CPUID flags: sse2 ssse3 sse41 sse42 avx2 avx512f rdtsc bmi1 bmi2 adx aes_ni clmul rdrand rdseed`)

--- a/README.md
+++ b/README.md
@@ -11,4 +11,9 @@ The nightly version of Rust is required to build this crate.
 
 To build everything, run tests and benchmarks, simply run `make`.
 
+If you have the `RUSTFLAGS` environment variable set, it will override Rust
+flags set in the repository's `.cargo/config`, so make sure you also add
+`-C target-feature=+aes,+ssse3` to your custom flags or the code will fail
+to build.
+
 [0]: https://sites.google.com/view/deoxyscipher

